controllers/users.controller.js

```diff
@@ -4,10 +4,14 @@ import passport from '../authentication/passport.js';
 import logger from '../config/logger/logger-service.js';
 
 export const UsersController = {
-  async login(ctx, next){
+  async login(req, res, next){
     await passport.authenticate('local', function (err, user) {
+      if (err) {
+        return next(err);
```

Comment:
why are you using in some placec "next(err)" and "return next(err)" in others?

Reply:
Added return for all next()-s